cityofaustin/atd-vz-data
# Create MVP of FTT Index

## Associated issues
Closes cityofaustin/atd-data-tech/issues/11190

We decided to split the above issue into multiple issues due to it's complexity, so this PR is primarily responsible for the front end UI of the FTT index and the initial creation of the `fatalities` table in the DB.

I populated the table with mock data so it is **not** accurate and you will notice the year and the date columns don't match up; that's fine I just wanted to be able to see what the table looks like with data and test pagination.

## Testing
**URL to test:** 
https://deploy-preview-1164--atd-vze-staging.netlify.app/#/fatalities

**Steps to test:**
1. Go to the new Fatalities section in the Nav bar, it is only visible to Admin access & above.
2. The following columns should be displayed in the table: Year, Crash ID, Case ID, Law Enforcement #, YTD Fatal Crash, YTD Fatalities, Crash Date, Location, Victim Name, Current FRB Status, and FRB Recommendation
3. Set the results to 10 per page so you can test pagination.
4. Clicking on the crash ID will take you to that crash details page
5. Not focusing on implementation of search or sort at this time so you don't need to test that, though I did turn on sorting for most columns.

---
#### Ship list
